src/content/history/index.md

```diff
@@ -0,0 +1,169 @@
+---
+title: History of Ethereum
+description: A history of the Ethereum blockchain including major milestones, releases, and forks.
+lang: en
+sidebar: true
+sidebarDepth: 2
+---
+
+# The history of Ethereum {#the-history-of-ethereum}
+
+A timeline of all the major milestones, forks, and updates to the Ethereum blockchain.
+
+<ExpandableCard title="What are forks?" contentPreview="Changes to the rules of the Ethereum protocol which often include planned technial upgrades.">
+Forks are when major technical upgrades or changes need to be made to the network – they often action [Ethereum Improvement Proposals (EIPs)](/eips) and change the "rules" of the protocol. When upgrades are needed in traditional, centrally-owned software, the company will just publish a new version for the end-user. Blockchains work differently because there is no central ownership.
```

Comment:
Noticed this link isn't rendering correctly.

Reply:
Yeah I've fixed this locally. Gotta leave a space between the component and the markdown and it renders.